bcda/auth/middleware.go

```diff
@@ -94,7 +99,8 @@ func RequireTokenACOMatch(next http.Handler) http.Handler {
 			if uuid.Equal(uuid.Parse(aco), uuid.Parse(string(urlUUID))) {
 				next.ServeHTTP(w, r)
 			} else {
-				http.Error(w, http.StatusText(http.StatusNotFound), http.StatusNotFound)
+				oo := responseutils.CreateOpOutcome(responseutils.Error, responseutils.Exception, "", responseutils.TokenErr)
+				responseutils.WriteError(oo, w, http.StatusUnauthorized)
```

Comment:
It's true that the user is unauthorized, but I'd lean toward keeping this as a 404. My concern is that a request from someone with the wrong credentials would show that the file does exist by giving a 401 rather than a 404. (Like how S3 handles it, if I remember correctly.) @rnagle or @msnook, what do you think?

Reply:
I agree with this, 404 is safer

Reply:
Agree.  I think we should keep this as 404.